pkg/lib/cockpit-components-dialog.jsx

```diff
@@ -138,12 +61,12 @@ class DialogFooter extends React.Component {
         let wait_element;
         let actions_disabled;
         let cancel_disabled;
-        if (this.state.action_in_progress) {
+        if (this.props.action_in_progress) {
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19200-20230821-124816-2fdc0a03-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)